modeling/algorithm/local-slice-algorithm_test.go

```diff
@@ -24,6 +24,159 @@ import (
 
 func TestLocalAlgorithm(t *testing.T) {
 	testCases := []algTestCase{
+		{
+			name: "corner case 1",
```

Comment:
add a bit more info for why this is/was a corner case

Reply:
changed case name